lib/src/model.dart

```diff
@@ -1227,7 +1236,10 @@ class Constructor extends ModelElement
   }
 
   @override
-  String get fullyQualifiedName => '${library.name}.$name';
+  String get fullyQualifiedName {
```

Comment:
consider a ternary and the short form?

```dart
String get fullyQualifiedName => isDefaultConstructor ? super.fullyQualifiedName : '${library.name}.$name';
```

Reply:
ternary makes this line a bit long for me.